testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/jsf/web.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright The WildFly Authors
+  ~ SPDX-License-Identifier: Apache-2.0
+  -->
+<web-app xmlns="https://jakarta.ee/xml/ns/jakartaee"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/web-app_6_0.xsd"
+         version="6.0">
+
+    <distributable/>
+
+    <!-- Make JSF parameters explicit for clustering and overly graceful -->
+
+    <context-param>
+        <param-name>javax.faces.STATE_SAVING_METHOD</param-name>
+        <param-value>server</param-value>
+    </context-param>
+    <context-param>
+        <param-name>com.sun.faces.serializeServerState</param-name>
+        <param-value>true</param-value>
```

Comment:
We never want this enabled.

Reply:
@pferraro OK I see I forgot to explain the motivation for this: this is meant to further isolate the problem being investigated. If this continues to fail, we can (mostly) rule out protostream and LRU view cache.

Nevertheless, added the a comment note about this, that this should be disabled.